### PR TITLE
Add @nilswellhausen's genome structure figure

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

WOW! @nilswellhausen once again made an amazing figure. This is my attempt to add it into the manuscript, although I forget how to do this every time, so it might need some debugging.

### Related issues

#745 

### Suggested reviewers (optional)

@ajlee21 @nilswellhausen 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by rando2** [comment]
Finally the figure it showing up correctly! I drafted a caption but would love feedback to improve it.

File: content/07.pathogenesis.md [hunk 1]
```diff
@@ -55,13 +55,19 @@ They are non-segmented, which means the viral genome is contained in a single ca
...
 The remaining third of the genome encodes structural proteins, including the spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins.
-Additional accessory genes are sometimes present between these two regions, depending on the species or strain. <!-- TODO: Add a figure showing the structure of the genome.-->
+Additional accessory genes are sometimes present between these two regions, depending on the species or strain.
 Much attention has been focused on the S protein, which is a critical structure involved in cell entry.
 
+![**Structure of SARS-CoV-2 capsid and genome.**
+A) The genomic structure of coronaviruses is highly conserved, including the order and organization of genes such as the spike (S), envelope (E), membrane (M) and nucleocapsid (N) proteins.
+B) The physical structure of the coronavirus virion, including the components encoded by conserved genes S, E, M and N. 
+](images/genome-structure.png)
+{#fig:genome-structure secno=1}
```
Comment (agitter) [inline content/07.pathogenesis.md:69]:
```suggestion
](images/genome-structure.png){#fig:genome-structure secno=1}
```

Something still isn't working with this figure, but I don't know what's wrong.  This is a shot in the dark.

I also noticed pandoc-fignos is complaining that fig:csse-deaths and fig:overview are defined multiple times.  I don't think that is causing this issue but will need to be addressed separately.

**Comment by rando2** [comment]
Thanks @agitter! It was working briefly and then I thought I broke it by adding the caption in the wrong place.
For the other two figures, at some point we should just delete 02.introduction.md and 50.discussion.md from master, and I assume that might help! I've been hesitant to do it because it's helpful to have them on master for figuring out who contributed what text.

**Comment by agitter** [comment]
Yeah, I agree it will be helpful to keep those extra markdown files around to track contributions.  Perhaps we could comment out the lines with the figures?

**Review by ajlee21 — APPROVED** [review]
This figure is AMAZING!

**Comment by agitter** [comment]
The last edit did the trick.  Apparently the curly braces need to be on the same line.  Here's a snapshot of the PDF:
image

This really is a fantastic figure.  Great work @nilswellhausen!

**Review by agitter — APPROVED** [review]

**Comment by rando2** [comment]
Thank you so much @agitter!

**Pull request merged**